From e1f2028e24fd52970fa0f14e3701b38ca16ccf5f Mon Sep 17 00:00:00 2001
From: Laura Lawrence <Laura.Lawrence@freescale.com>
Date: Thu, 12 Jun 2008 14:10:21 -0500
Subject: [PATCH] ENGR00079742 Fix Audio Crackling Noise in 16-bit and 32-bit mode

Enlarge Audio IRAM Buffer to 24K to fix noise during H.264 decode
Observed in both 16-bit and 32-bit mode

Signed-off-by: Laura Lawrence <Laura.Lawrence@freescale.com>
---
 include/asm-arm/arch-mxc/mx37.h |    2 +-
 1 files changed, 1 insertions(+), 1 deletions(-)

diff --git a/include/asm-arm/arch-mxc/mx37.h b/include/asm-arm/arch-mxc/mx37.h
index 6c1ed02..d60fc7c 100644
--- a/include/asm-arm/arch-mxc/mx37.h
+++ b/include/asm-arm/arch-mxc/mx37.h
@@ -74,7 +74,7 @@
 #define CONFIG_SDMA_IRAM_SIZE 0
 #endif
 #ifdef CONFIG_SND_MXC_SOC_IRAM
-#define SND_RAM_SIZE 0x3000
+#define SND_RAM_SIZE 0x6000
 #else
 #define SND_RAM_SIZE 0
 #endif
-- 
1.5.4.4

